test/test_connection_pool.rb

```diff
@@ -179,7 +186,7 @@ def test_with_timeout_override
       pool.with { |net| net.do_something }
     end
 
-    pool.with(:timeout => 0.1) do |conn|
+    pool.with(:timeout => 2 * NetworkConnection::SLEEP_TIME) do |conn|
```

Comment:
I don't this all the tests necessarily want to have the same timeout, it's just a coincidence that they do today